datalad_next/runners/data_processors/decode.py

```diff
@@ -0,0 +1,86 @@
+""" This module contains data processors for the data pipeline processor
+
+The data processors contained here are:
+
+  - decode_utf8_processor
+
+"""
```

Comment:
None of this documentation is accessible in the renderer docs. This hides the fact that all the `data_processors` modules are documented in the exact same way, making them indistinguishable from each other.

I think the pattern to manually list the main components in a module in the top-level docstring is not optimal. In the renderer documentation this would e automatic/obvious already. Here it adds content that needs to be kept in sync. I believe an `__all__` declaration would be a more appropriate alternative.

In general I think that that sorting of code in all the modules is artificially creating the need to describe what matters most. The "public" component tends to be last in the file (although it could be first).